src/reducers/picklist/index.js

```diff
@@ -53,11 +60,44 @@ const clearItems = (type) => (state) => ({
   [type]: []
 });
 
+export const getPath = (pathStr) => {
+  const pathStrArr = pathStr.substr(1, pathStr.length - 2).split(',')
+  // EX: ,1,2,3,19, will be transformed to 1,2,3,19
```

Comment:
we need to correct this comment to:
 // EX: ,1,2,3,19, will be transformed to 1,2,3,